back/db/migrate/20200527094026_add_ordering_to_projects_topics.rb

```diff
@@ -1,10 +1,21 @@
 class AddOrderingToProjectsTopics < ActiveRecord::Migration[6.0]
-  def change
-    add_timestamps :projects_topics
-    add_column :projects_topics, :ordering, :integer
+  class StubProjectsTopic < ActiveRecord::Base
+    self.table_name = 'projects_topics'
+  end
+
+  reversible do |dir|
+    dir.up do
+      add_timestamps :projects_topics
+      add_column :projects_topics, :ordering, :integer
+
+      StubProjectsTopic.order(:created_at).each.with_index do |projects_topic, index|
+        projects_topic.update_column :ordering, index
+      end
+    end
 
-    ProjectsTopic.order(:created_at).each.with_index do |projects_topic, index|
-      projects_topic.update_column :ordering, index
+    dir.down do
```

Comment:
The main advantage of reversible is built-in rails migration methods will continue to work in both directions (migrate and rollback). So, it can look like that.

```ruby
def change
  add_timestamps :projects_topics
  add_column :projects_topics, :ordering, :integer

  reversible do |dir|
    dir.up do
      StubProjectsTopic.order(:created_at).each.with_index do |projects_topic, index|
        projects_topic.update_column :ordering, index
      end
    end
  end 
end
``` 

So, the original code is almost untouched.